featuretools/entityset/entity.py

```diff
@@ -82,6 +83,11 @@ def __init__(self, id, df, entityset, variable_types=None,
 
         self.set_secondary_time_index(secondary_time_index)
 
+        # Fill in single `NaN` values in LatLong variables with a tuple
```

Comment:
perhaps this conversion should happen somewhere in the logic of `Entity._create_variables`

do we want to alert the user about their input data being less than ideal?

Reply:
I wasn't sure where this should reside exactly, but moving to `_create_variables` seems fine.

I also think it would make sense to warn if we perform this operation on a column, especially since we are modifying the entity dataframe.